databuilder/databuilder/models/description_metadata.py

```diff
@@ -0,0 +1,150 @@
+# Copyright Contributors to the Amundsen project.
```

Comment:
Separated `DescriptionMetadata` from `TableMetadata` due to circular import issues with `TypeMetadata`